packages/big-design/src/components/TableNext/SelectAll/helpers.ts

```diff
@@ -1,23 +1,12 @@
-import { TableExpandable, TableSelectable } from '../types';
+import { getPagedIndex } from '../helpers';
+import { TableExpandable, TablePaginationProps, TableSelectable } from '../types';
 
-interface SelectAllRowsArg<T> {
-  isExpandable: boolean;
-  items: T[];
-  selectedItems: TableSelectable['selectedItems'];
-  expandedRowSelector?: TableExpandable<T>['expandedRowSelector'];
-}
+import { SelectAllProps } from './SelectAll';
 
-export function getTotalSelectedItems<T>(
-  items: T[],
-  selectedItems: TableSelectable['selectedItems'],
-) {
-  return items.reduce((acc, _parentRow, parentRowIndex) => {
-    if (selectedItems[parentRowIndex] !== undefined) {
-      return acc + 1;
-    }
+type SelectAllRowsArg<T> = Omit<SelectAllProps<T>, 'onChange'>;
 
-    return acc;
-  }, 0);
+export function getTotalSelectedItems(selectedItems: TableSelectable['selectedItems']) {
```

Comment:
looks like this changing from getting 'total parents' to 'total parents and children' ? Do we want to include children in total count?

Reply:
Ah you're right, good catch!